carbonmark/components/ProjectFilterModal/index.tsx

```diff
@@ -85,69 +92,100 @@ export const ProjectFilterModal: FC<ProjectFilterModalProps> = (props) => {
     value: vintage,
   }));
 
+  const fetchProjects = (values: ModalFieldValues) => {
+    const { search, sort } = router.query;
+    const query = search ? { ...values, search, sort } : { ...values, sort };
```

Comment:
Isn't this equivalent? (unless the values are possibly null?)
 
```suggestion
    const query = { ...values, search, sort };
```

Reply:
Hmm, I thought I had a case where the search param was getting added as an empty string to the url hence the check. On second look now it seems this isn't the case, so will remove and clean this up 👍